README.md

```diff
@@ -59,8 +60,9 @@ dsm_agent_download_port | Int | The port to connect to the Deep Security Manager
 dsm_agent_activation_hostname | String | The hostname for the agents to communicate with once deployed. For Marketplace and software deployments this is typically the same hostname as 'dsm_agent_download_hostname'. | agents.deepsecurity.trendmicro.com
 dsm_agent_activation_port | Int | The port to use for the agent heartbeat (the regular communication). For Marketplace and software deployments, the default is 4120. | 443
 tenant_id | String | In a multi-tenant installation (like Deep Security as a Service), this identifies the tenant account to register the agent with. |
-token | String | In a multi-tenant installation (like Deep Security as a Service), this identifies the tenant account to register the agent with. |
+token/tenant_password | String | In a multi-tenant installation (like Deep Security as a Service), this identifies the tenant account to register the agent with. In latest Deep Security Manager, "tenant_password" has been replaced by "token". Still keep parameter "tenant_password" for backward compatible |
```

Comment:
In the latest Deep Security Manager, "tenant_password" has been replaced with "token".  "tenant_password" has been kept for backwards compatibility.